doc/report.Rmd

```diff
@@ -53,26 +55,26 @@ Each point in which a conditional statement is evaluated and the data could pote
 Splits are typically evaluated by Gini impurity or entropy.
 
 $$
-Gini\ Impurity\ =\ I_G(t)\  = 1 - \sum_{i=1}^{C}p(i|t)^2
+\text{Gini Impurity} =\ I_G(t)\  = 1 - \sum_{i=1}^{C}p(i|t)^2
 $$
 
 $$
-Entropy\ =\ I_H(t)\ = -\sum_{i=1}^{C}p(i|t)log_2p(i|t)
+\text{Entropy} =\ I_H(t)\ = -\sum_{i=1}^{C}p(i|t)log_2p(i|t)
 $$
 
-Where $p(i|t)$ is the proportion of samples that belong to class $i$ for a particular node t.
+Where $p(i|t)$ is the proportion of samples that belong to class $i$ for a particular node $t$.
 
 A Gini Impurity of 0 or an entropy value of 0 indicates a completely homogeneous group, which cannot be improved upon. A split doesn’t need to have a value of 0 to create the best node though, these metrics are used for comparison. All possible different variables and different cutoffs within those variables are all tried and their Gini impurity or entropy calculated, then, the one with the best (lowest) value is the split used at that node. This is done recursively at each node until all possible splits no longer offer a decrease in the metric of choice. The resulting collection of nodes is a decision tree. 
 
 Decision trees perform quite poorly with new samples. Including a threshold value for the accuracy metric can help, as it keeps trees more simple, but they are still prone to overfitting. Collecting them into a random forest addresses this issue. True to its name, a random forest is a collection of decision trees that have a component of randomness in their creation. The predictive aspect of a random forest doesn’t only rely on one tree, it is determined by totaling up the decisions, or “votes,” that all of the trees make.
 
 To create each tree the first step is bootstrapping the training dataset. The decision tree is then created with that bootstrapped data, but with slightly different rules. For each node, only a subset of the variables are randomly selected and used as candidates to split up the bootstrapped data. Not all the variables are used at each node because doing so reduces correlation and introduces randomness into the model. Whichever variable best splits the data will be the variable that is kept at that node in that specific tree. This is done recursively until no more splits are beneficial, just as in a regular decision tree. To create the next tree the process is the same, but starts completely over with a new bootstrap sample of the training data. This is repeated for a chosen number of trees.
```

Comment:
"This is done recursively until no more splits are beneficial, just as in a regular decision tree."  True, only if minimum size is not reached.